Cython/Compiler/ExprNodes.py

```diff
@@ -7185,6 +7185,74 @@ def analyse_as_python_attribute(self, env, obj_type=None, immutable_obj=False):
                 error(self.pos,
                       "Object of type '%s' has no attribute '%s'" %
                       (obj_type, self.attribute))
+        else:
+            self.python_attribute_lookup_warnings(env)
```

Comment:
The `if not … and not …` above feels like we should swap the if-clause with the else.
Also, we probably shouldn't get here for the `is_error` case either, right?